### ISSUE TITLE
"Overlap" not calculated correctly on in party_polarization article

### ISSUE THREAD

**Issue by JeffreyBLewis** [voteview/articles#12]
John G. Bullock pointed out that  
```{r}
      overlap = (sum(nominate_dim1[party_code==200] <
                       max(nominate_dim1[party_code==100],na.rm=T),na.rm=T)  +
                 sum(nominate_dim1[party_code==100] >
                       max(nominate_dim1[party_code==200],na.rm=T),na.rm=T))/
                 (sum(!is.na(nominate_dim1[party_code==100]))+
                  sum(!is.na(nominate_dim1[party_code==200]))),
```
should be 

```{r}
  overlap = (sum(nominate_dim1[party_code==200] <
                       max(nominate_dim1[party_code==100],na.rm=T),na.rm=T)  +
                 sum(nominate_dim1[party_code==100] >
                       min(nominate_dim1[party_code==200],na.rm=T),na.rm=T))/
                 (sum(!is.na(nominate_dim1[party_code==100]))+
                  sum(!is.na(nominate_dim1[party_code==200]))),
```

**Comment by JeffreyBLewis** [comment]
FIxing in #13

**Comment by JeffreyBLewis** [comment]
Fixed in #13.
